### PR TITLE
Clarify how to contribute and point out github registry pitfall

### PR DESCRIPTION
**Pull request by littldr**
Hi!

after falling into the trap myself (https://github.com/pybricks/support/issues/174), I propose to move the `Github Packages` section to the top of the `CONTRIBUTING.md` and add a note that this step is really really necessary. 😅 

Additionally I added a hint in the `README.md` pointing to the `CONTRIBUTING.md` file. Maybe this will help others to read the `CONTRIBUTING.md` more careful than I did. :-)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,6 +4,28 @@
 
 These are the tools you will need to build and run `pybricks-code` locally.
 
+## Github Packages
```
Comment (dlech) [inline CONTRIBUTING.md:7]: To make it even more obvious, we could change the title to `Important Note about Github Packages`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,6 +4,28 @@
...
+
+> This step is mandatory and cannot be skipped. Otherwise you will have no access to some referenced dependencies in [package.json](./package.json)!
```
Comment (dlech) [inline CONTRIBUTING.md:9]: How about `**bold**` instead of `> quote`?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -70,20 +69,18 @@ where you would like to save the source code and run:
...
-probably skipped the logging into the Github package registry as described above.
+probably skipped the logging into the Github package registry as [described above](#github-packages).
 
 # Software Stack
 
 This project was bootstrapped with [Create React App][create-react-app].
 
 [create-react-app]: https://github.com/facebook/create-react-app
 
-
```
Comment (dlech) [inline CONTRIBUTING.md:81]: Why removing the extra newlines?

**Review by dlech — COMMENTED** [review]
Thanks for this. I just have a few suggestions (see inline comments).

File: CONTRIBUTING.md (hunk 3 above)
Comment (littldr) [reply CONTRIBUTING.md:81]: Just for consistency. The first couple sections had no double newlines between each other and as they make no difference in the rendered version I thought of removing them.

But I can re-add them if they were added intentionally.

**Comment by dlech** [comment]
Thanks!

**Pull request merged**